html/elements/multicol.json

```diff
@@ -0,0 +1,59 @@
+{
+  "html": {
+    "elements": {
+      "multicol": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Element/multicol",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "Up to Firefox 21 included, though having no effect, this element implemented the &lt;HTMLSpanElement&gt; instead of the compliant &lt;HTMLUnknowElement&gt;."
```

Comment:
I would put this as `"Up to Firefox 22, though not supported, a this element was associated with the <code>HTMLSpanElement</code> interface. It was then fixed and is now associated with the <code>HTMLUnknownElement</code> interface as requested by the specification."` (same for firefox_android)

Reply:
Fixed.